### ISSUE TITLE
send last known location if lost fix

### ISSUE THREAD

**Issue by FreshAF123** [tananaev/traccar-client-android#380]
isit possible to send the last known location if fix has been lost? therefore at every frequency we are able to get a location update

**Comment by oliv3** [comment]
Hi, @FreshAF123. From your questions I understand you have concerns with location updates frequency. What exactly do you want to achieve ? What are your goals/wishes ?

**Comment by oliv3** [comment]
This doesn't answer my question; why would you want to send an outdated location ?
Device might actually be far from it's last known location.

**Comment by Tyler-2** [comment]
Agreed with oliv3: The only way I would want old locations to be sent is if they were timestamped with the old timestamp. This would essentially be a "ping" to the server, which might be useful in some way to establish that the client is online but has an unknown position... But I'm not clear on the specific use case.

**Comment by AUSITStuff** [comment]
Our problem has been if we want to use the SOS feature, when using High Accuracy it works fine if you are outdoors however once it loses GPS lock indoors the app essentially goes offline. At least with a last known location we would know where to start looking.

Essentially it should be in the Unknown category in that it is reporting to server but cannot guarantee accuracy but at least that way the SOS widget would still function.
